changelog.d/2283.fixed.md

```diff
@@ -0,0 +1 @@
+Fixed a bug where non-existent hosts in outgoing filter would prevent the application from making initiating outgoing connections.
```

Comment:
> from making initiating outgoing connections

"making initiating" sounds a bit weird? Maybe it's just me though.

Reply:
Ah sorry, should be just "initiating"